modules/auxiliary/scanner/http/wordpress_cp_calendar_sqli.rb

```diff
@@ -0,0 +1,74 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'uri'
+require 'msf/core'
+
+class Metasploit4 < Msf::Auxiliary
+
+  include Msf::Exploit::Remote::HttpClient
```

Comment:
It technically does exploit the vuln in order to check the validity of the vuln :)

It also isn't a wordpress vuln per se, just a plugin for wordpress, the version of wordpress itself doesn't matter.

In any case, I agree, the scanner is essentially its own check method.